src/components/modals/SavedViewsModal.svelte

```diff
@@ -0,0 +1,142 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import type { ICellRendererParams } from 'ag-grid-community';
+  import { createEventDispatcher } from 'svelte';
+  import { initializeView, view, views } from '../../stores/views';
+  import type { DataGridColumnDef } from '../../types/data-grid';
+  import type { View } from '../../types/view';
+  import effects from '../../utilities/effects';
+  import { setQueryParam } from '../../utilities/generic';
+  import BulkActionDataGrid from '../ui/DataGrid/BulkActionDataGrid.svelte';
+  import DataGridActions from '../ui/DataGrid/DataGridActions.svelte';
+  import Modal from './Modal.svelte';
+  import ModalContent from './ModalContent.svelte';
+  import ModalFooter from './ModalFooter.svelte';
+  import ModalHeader from './ModalHeader.svelte';
+
+  export let height: number | string = 150;
+  export let width: number | string = 380;
+
+  type CellRendererParams = {
+    deleteView: (view: View) => void;
+    openView: (view: View) => void;
+  };
+  type ViewCellRendererParams = ICellRendererParams<View> & CellRendererParams;
+
+  const dispatch = createEventDispatcher();
+  const columnDefs: DataGridColumnDef[] = [
+    {
+      field: 'id',
+      filter: 'number',
+      headerName: 'ID',
+      resizable: true,
+      sortable: true,
+      suppressAutoSize: true,
+      suppressSizeToFit: true,
+      width: 60,
+    },
+    { field: 'name', filter: 'text', headerName: 'Name', resizable: true, sortable: true },
+    { field: 'owner', filter: 'text', headerName: 'Owner', resizable: true, sortable: true },
+    { field: 'updated_at', filter: 'text', headerName: 'Last Edited', resizable: true, sortable: true },
```

Comment:
Might consider chopping off some of the precision on this field? Feels like we only need up to seconds at the very most. Might also help with being able to see the whole date string
<img width="271" alt="image" src="https://user-images.githubusercontent.com/4419607/216725119-52c200e7-609d-43d2-8c52-7524e46bf267.png">